.werft/vm/manifests.ts

```diff
@@ -334,8 +339,8 @@ write_files:
       kubectl apply -f /var/lib/gitpod/manifests/metrics-server.yaml
 
       # install CSI snapshotter CRDs and snapshot controller
-      kubectl apply -f /var/lib/gitpod/manifests/csi-snapshotter-crd.yaml
-      kubectl apply -f /var/lib/gitpod/manifests/csi-snapshot-controller.yaml
+      kubectl apply -f /var/lib/gitpod/manifests/csi-driver.yaml
+      kubectl apply -f /var/lib/gitpod/manifests/csi-config.yaml
```

Comment:
These paths got changed in https://github.com/gitpod-io/gitpod-packer-gcp-image/pull/114